src/ui/gcloud/gcloud.js

```diff
@@ -199,22 +214,97 @@ cwc.ui.GCloud.prototype.selectBucketDialog = function(items) {
   }
 };
 
+cwc.ui.GCloud.prototype.selectBucket = function(bucketId) {
+  this.bucketName = bucketId;
+  var foldersContainer = goog.dom.getElement(this.prefix + 'folders-container');
+  var folderPicker = goog.dom.getElement(this.prefix + 'folders-picker');
+  var folderNav = goog.dom.getElement(this.prefix + 'folders-nav');
+  var storageButtonEvent = (function(event) {
+    var sibling = goog.dom.getNextElementSibling(event.target);
+    if (sibling != null) {
+      this.storagePrefix = event.target['gcloud-folders-prefix'];
+      var nextSibling = goog.dom.getNextElementSibling(sibling);
+      while (nextSibling != null) {
+        goog.dom.removeNode(sibling);
+        sibling = goog.dom.getNextElementSibling(nextSibling);
+        goog.dom.removeNode(nextSibling);
+        nextSibling = null;
+        if (sibling != null) {
+          nextSibling = goog.dom.getNextElementSibling(sibling);
+        }
+      }
+      this.currentDirectory();
+    }
+  }).bind(this);
+  var rootButton = goog.dom.createDom(
+      'button', {'gcloud-folders-prefix': ''}, '/');
+  folderNav.append(rootButton);
+  goog.events.listen(
+      rootButton, goog.events.EventType.CLICK, storageButtonEvent, false, this);
+  var selectFolderEvent = (function(event) {
```

Comment:
Use arrow function instead, because project is ES6 enabled.
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions